cli/ClassifyTestCasesCommand.php

```diff
@@ -0,0 +1,134 @@
+<?php
+declare(strict_types=1);
+
+namespace UAClassifier\Cli;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Yaml\Yaml;
+
+class ClassifyTestCasesCommand extends Command
+{
+    /**
+     * @var array $testCaseData Array to append test case data to before parsing to YAML
+     */
+    protected $testCaseData = [];
+
+    /**
+     * @var array $classificationMap
+     */
+    protected $classificationMap = [
+        'm' => 'Mobile',
+        't' => 'Tablet',
+        'd' => 'Desktop',
+        's' => 'Spider'
+    ];
+
+    /**
+     * @var array $targetMap
+     */
+    protected $targetMap = [
+        'u' => 'User Agent',
+        'o' => 'Operating System',
+        'd' => 'Device'
+    ];
+
+    /**
+     * @inheritdoc
+     */
+    protected function configure(): void
+    {
+        $this->setName('classify-test-cases')
+             ->setDescription('Update test-case data with prompted device classification');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $io = new SymfonyStyle($input, $output);
+        $io->section('Classification Configuration');
+
+        $classification = $io->choice('Select classification to grant', $this->classificationMap);
+
+        $target = $io->choice('Apply classification by', $this->targetMap);
+
+        $regex = $io->ask("Specify regex to classify records by", null, function($value) {
+            if (!preg_match('/^\/[\s\S]+\/[gmixXsuUAJD]?$/', $value)) {
```

Comment:
`\s` is the opposite of `\S`, so these two cover every character. At which point, it seems simpler to just allow `.+`